sdk/core/Azure.Core.Experimental/src/DynamicTableEntity.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.IO;
+using Azure.Data.Tables;
+
+// TODO: remove
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+
+namespace Azure.Core.Dynamic
+{
+    /// <summary>
+    /// Dynamic layer over Azure.Tables.TableEntity.
+    /// </summary>
+    public class DynamicTableEntity : DynamicData
```

Comment:
This seems quite specific to be in Azure.Core.   Just a temporary stop before moving to the Tables package or are we thinking it'll live here forever?

Reply:
Yes, it's just a proof of concept for this prototype - to show that DynamicData can be generalized to non-JSON types.  If we went with this approach, Krzysztof might put something like this into the CloudMachine code base.